css/port.css

```diff
@@ -0,0 +1,297 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;1,100;1,200;1,300;1,500;1,600;1,700&display=swap');
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: 'Poppins', sans-serif;
+  background-color: #fff;
+}
+
+.bk_container {
+  background-color: #e5e5e5;
+  max-width: 375px;
+  margin: auto;
+}
+
+.port_container {
+  max-width: 375px;
```

Comment:
- [ ] You should remove the `max-width` because it is making your website not responsive.